rollbar_dart/test/client_server_utils.dart

```diff
@@ -20,42 +20,42 @@ Sender createTextSender(Config c) {
 /// Useful class to do some basic tests across isolates, without setting up a full blown
 /// HTTP server. It accepts and collects raw text over a TCP socket, 1 message per line.
 class RawTextSocket {
-  Isolate _isolate;
-  int _port;
-  ReceivePort _receivePort;
+  late Isolate _isolate;
+  int? _port;
+  late ReceivePort _receivePort;
 
   RawTextSocket._();
 
-  String get endpoint => '$PREFIX$port';
+  String get endpoint => '$endpointPrefix$port';
 
-  Future<int> _start() async {
+  Future<int?> _start() async {
     _receivePort = ReceivePort();
 
     var tcpInfoPort = ReceivePort();
 
     _isolate = await Isolate.spawn(
         _server, [tcpInfoPort.sendPort, _receivePort.sendPort]);
 
-    int port = await tcpInfoPort.first;
+    var port = await (tcpInfoPort.first);
```

Comment:
Let's skip parentheses, they seem to be redundant

```suggestion
    var port = await tcpInfoPort.first;
```